### ISSUE TITLE
Change of colours in some components between version 104.2.0 and 104.26.1

### ISSUE THREAD

**Issue by arvig33** [Sage/carbon#4900]
### Current behaviour

**- Toggle button**
before: 
![image](https://user-images.githubusercontent.com/17083504/155705443-63ef3bfc-0c5f-428d-b62c-65e0ffdb94f6.png)
![image](https://user-images.githubusercontent.com/17083504/155705756-030eb9ba-7482-4c6f-b378-6fa5144b48bb.png)

now: 
![image](https://user-images.githubusercontent.com/17083504/155706001-06719c50-a5af-4c93-9b16-b3a8dcb7b068.png)
![image](https://user-images.githubusercontent.com/17083504/155705915-d70b9728-15f5-4d2c-9214-1e754ff6e584.png)

https://carbon.sage.com/?path=/story/button-toggle-group--controlled

**- Icon on Flat table**
before:
![image](https://user-images.githubusercontent.com/17083504/155706320-c45ec5c3-422a-4964-b79e-843eede97f17.png)

now: 
![image](https://user-images.githubusercontent.com/17083504/155706219-cccbd6dd-5c85-418c-b48d-d4120947c41a.png)

https://carbon.sage.com/iframe.html?id=flat-table-expandable--keyboard-accessible-subrows&viewMode=docs#with-row-headers-with-custom-paddings

### Expected behaviour

The colours do not seem to be those intended.

### CodeSandbox or Storybook URL

https://carbon.sage.com/iframe.html?id=flat-table-expandable--keyboard-accessible-subrows&viewMode=docs#with-row-headers-with-custom-paddings

### Carbon Version

104.26.1

### What browsers are you seeing the problem on?

Chrome

### What Operating System are you seeing the problem on?

Windows

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by jamime** [comment]
Based on the [RFC](https://github.com/Sage/carbon/blob/master/rfcs/text/design-tokens.md) I had interpreted this as the existing `mint` and `aegean` would have no visual changes when introducing design tokens. 

@jhajduk-sage please can you confirm if this change was intentional (introduced in https://github.com/Sage/carbon/pull/4584).

**Comment by jhajduk-sage** [comment]
Yes, the change is intentional. Tokens are applied properly according to design guidelines. The resulting differences were tradeoffs of migration to css variables, applying tokens and rebranding.

In most cases there are no visual changes, however there are few places where such tradeoffs were necessary.
